### DISCUSSION TITLE
更新2.6.3版本后，阅读进度和高能进度条丢失

### DISCUSSION THREAD

**Discussion by Shuchun-Wang** [MuiseDestiny/zotero-style#385] [Q&A] [answered]
更新2.6.3版本后，高能进度条丢失，
再且重新看文献也不显示进度，回退到之前版本也无法解决。
尝试过#232万能解决方法，也无法解决。
![image](https://user-images.githubusercontent.com/77203000/233826003-7e004882-2000-4f6a-8b29-0bcaf95f0673.png)

求帮助，我的zotero不能离开进度条

**Comment by MuiseDestiny** [comment]
只是不显示了，禁用其他插件，或者disable 后enable

**Reply by Shuchun-Wang** [reply]
亲爱的开发者，
       我disable了其他所有的插件，又重启了好几次zotero，进度条还是不能显示，
       又把zotero disable 后enable了好几次，结果还是不行。
       盼指导。

**Reply by MuiseDestiny** [reply]
todesk控制发muisedestiny@gmail.com，明天的吧

**Reply by Shuchun-Wang** [reply]
好的，谢谢

**Reply by Shuchun-Wang** [reply]
![image](https://user-images.githubusercontent.com/77203000/233831674-06c47a72-5cc7-44d8-9c84-2c0e4ed1cdd3.png)
您好，
最新的情况是，安装2.6.3版本之后阅读的进展有了，但是今天之前看的所有文献进度条全没有了，请问这种情况应该怎么设置呢？

**Reply by MuiseDestiny** [reply]
你是从什么版本安装的最新版，就是说昨天之前你用的什么版本

**Reply by Shuchun-Wang** [reply]
您好，
zotero版本6.0.26，最近一直没有更新过
![image](https://user-images.githubusercontent.com/77203000/233832986-82de6918-d954-472a-b07d-02ed1c60a16b.png)

**Reply by MuiseDestiny** [reply]
插件呢

**Reply by Shuchun-Wang** [reply]
插件的版本是1.4.7

**Reply by MuiseDestiny** [reply, marked as answer]
迁移，issue里搜一下

**Reply by Shuchun-Wang** [reply]
谢谢大佬，成功了！我的进度回来了！

**Reply by MuiseDestiny** [reply]
ok
